### PR TITLE
TSS Landing page (first iteration)

### PR DESCRIPTION
**Pull request by john12321**
The aim of this ticket is to rapidly develop a usable landing page to hopefully improve UX and allow us to work on the DSP FE work at the earliest opportunity. We can then look at making improvements to the landing page and associated components alongside the development of the DSP functionality. 

- Add a new Home page (use the Digital service manual design).
- Separate out Profile comps to help with future development of landing page and DSP work (it also makes them more accessible from the new Home page ).
- Add simple Home link (the beginnings of a Breadcrumb comp) from all pages - apart from Home page and MFA page (when no MFA set).
- Improve routing so duff URL's get 'Page Not Found' msg (which also helps with follow-up work developing the Breadcrumb comp for nested routes).

Possible next steps (DSP work permitting)
1. Breadcrumb comp improvements
2. Improved messaging/ links to help user change their data

TIS21-3906

### CODE REVIEW

File: cypress/e2e/mfa/MFA.spec.ts [hunk 1]
```diff
@@ -8,10 +8,15 @@ describe("MFA set-up", () => {
...
+    cy.get('[data-cy="MFA"]').first().click();
+    cy.get('[data-cy="homeLink"]').should("exist");
+    cy.get(".nhsuk-fieldset__heading")
+      .should("exist")
+      .should("include.text", "MFA (Multi-Factor Authentication) set-up");
+
     cy.get("[data-cy=mfaAlreadyWarning]").should("exist");
     cy.get("[data-cy=mfaAlreadyText]")
-      .should("exist")
+      .should("exist")
```
Comment (Judge40) [inline cypress/e2e/mfa/MFA.spec.ts:19]: Random bit of whitespace if you feel like cleaning it up

File: cypress/component/main/Main.cy.tsx [hunk 2]
```diff
@@ -26,60 +26,63 @@ describe("Main", () => {
...
+  //   };
+  //   mount(
+  //     <Provider store={store}>
+  //       <Router history={history}>
+  //         <MockedMainFailed />
+  //       </Router>
+  //     </Provider>
+  //   );
+  //   cy.get("[data-cy=BtnMenu]").should("not.exist");
+  // });
```
Comment (Judge40) [inline cypress/component/main/Main.cy.tsx:87]: What's the future for these tests?

**Comment by ReubenRobertsHEE** [comment]
image
Looking pretty good to me!
Two minor points: (a) is it intentional that the text under 'Your post-graduate training programme resource' only spans 2/3 of the width? and (b) if it were me, I'd consider omitting the MFA and Support panels, but I don't want to start a debate if they were included in the mock-ups that this work is based-off 😁

File: components/home/HomeHeaderSection.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+const HomeHeaderSection = () => {
+  return (
+    <section className="nhsuk-hero">
+      <div className="nhsuk-width-container nhsuk-hero--border app-width-container">
+        <div className="nhsuk-grid-row">
+          <div className="nhsuk-grid-column-two-thirds">
```
Comment (ReubenRobertsHEE) [inline components/home/HomeHeaderSection.tsx:6]: `nhsuk-grid-column-three-thirds` stretches this out to fill the width 🤷

File: cypress/e2e/mfa/MFA.spec.ts (hunk 1 above)
Comment (john12321) [reply cypress/e2e/mfa/MFA.spec.ts:19]: Hmm prettier usually catches those on save. Cheers!

File: cypress/component/main/Main.cy.tsx (hunk 2 above)
Comment (john12321) [reply cypress/component/main/Main.cy.tsx:87]: Oops. Covered elsewhere now. Cheers!

**Comment by Judge40** [comment]
> image Looking pretty good to me! Two minor points: (a) is it intentional that the text under 'Your post-graduate training programme resource' only spans 2/3 of the width? and (b) if it were me, I'd consider omitting the MFA and Support panels, but I don't want to start a debate if they were included in the mock-ups that this work is based-off 😁

For the first iteration the cards mirroring the existing nav is fine in my opinion, it's likely to start looking quite different once we start building more stuff in anyway.

**Comment by john12321** [comment]
> image Looking pretty good to me! Two minor points: (a) is it intentional that the text under 'Your post-graduate training programme resource' only spans 2/3 of the width? and (b) if it were me, I'd consider omitting the MFA and Support panels, but I don't want to start a debate if they were included in the mock-ups that this work is based-off 😁

Thanks for your immediate feedback @ReubenRobertsHEE ! 😄 . Thinking behind the the 2/3 is to allow room for a quick notification/ call to action etc. Speaking to Naz , she had a couple of ideas so left some room. Second point, I refer to my honourable friend's reply to your comment 😀

File: components/home/HomeHeaderSection.tsx (hunk 3 above)
Comment (john12321) [reply components/home/HomeHeaderSection.tsx:6]: three-thirds goes slightly outside the container. Could remove the grid css altogether though if we want full container width?... You've twisted my arm @ReubenRobertsHEE  I'll push the change with full width header.  😀  
image

**Review by Judge40 — APPROVED** [review]

**Pull request merged**